hooks/pre/10-reset_data.rb

```diff
@@ -62,14 +41,13 @@ def empty_candlepin_database
 end
 
 def reset_candlepin
-  Kafo::KafoConfigure.logger.info 'Dropping Candlepin database!'
   commands = [
     'rm -f /var/lib/candlepin/cpdb_done',
     'rm -f /var/lib/candlepin/cpinit_done',
     'systemctl stop tomcat',
```

Comment:
We already have `stop_services` in the beginning. Is this still needed? (similar thing in reset_pulp`)